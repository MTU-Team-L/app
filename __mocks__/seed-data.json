{
  "cards": {
    "rows": [{"id":"Dungeon Master","key":"Dungeon Master","value":{"deleted":false,"rev":"1-bee88cce49934d0d9ba2a4df9e0274f9"},"doc":{"object":"card","id":"a4cf1827-2ce6-4592-b213-858dcb115ae0","oracle_id":"6bbd0bf6-cb22-431f-be1b-e76febf68cfc","multiverse_ids":[],"name":"Dungeon Master","lang":"en","released_at":"2017-09-20","uri":"https://api.scryfall.com/cards/a4cf1827-2ce6-4592-b213-858dcb115ae0","scryfall_uri":"https://scryfall.com/card/htr/2/dungeon-master?utm_source=api","layout":"normal","highres_image":true,"image_uris":{"small":"https://img.scryfall.com/cards/small/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783","normal":"https://img.scryfall.com/cards/normal/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783","large":"https://img.scryfall.com/cards/large/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783","png":"https://img.scryfall.com/cards/png/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.png?1573039783","art_crop":"https://img.scryfall.com/cards/art_crop/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783","border_crop":"https://img.scryfall.com/cards/border_crop/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783"},"mana_cost":"{2}{W}{U}","cmc":4,"type_line":"Legendary Planeswalker — Dungeon Master","oracle_text":"+1: Target opponent creates a 1/1 black Skeleton creature token with \"When this creature dies, each opponent gains 2 life.\"\n+1: Roll a d20. If you roll a 1, skip your next turn. If you roll a 12 or higher, draw a card.\n−6: You get an adventuring party. (Your party is a 3/3 red Fighter with first strike, a 1/1 white Cleric with lifelink, a 2/2 black Rogue with hexproof, and a 1/1 blue Wizard with flying.)","loyalty":"1d4+1","colors":["U","W"],"color_identity":["U","W"],"legalities":{"standard":"not_legal","future":"not_legal","historic":"not_legal","pioneer":"not_legal","modern":"not_legal","legacy":"not_legal","pauper":"not_legal","vintage":"not_legal","penny":"not_legal","commander":"not_legal","brawl":"not_legal","duel":"not_legal","oldschool":"not_legal"},"games":["paper"],"reserved":false,"foil":false,"nonfoil":true,"oversized":false,"promo":false,"reprint":false,"variation":false,"set":"htr","set_name":"2016 Heroes of the Realm","set_type":"memorabilia","set_uri":"https://api.scryfall.com/sets/e9dbe497-c76a-4037-82c1-7ef338d6c54c","set_search_uri":"https://api.scryfall.com/cards/search?order=set&q=e%3Ahtr&unique=prints","scryfall_set_uri":"https://scryfall.com/sets/htr?utm_source=api","rulings_uri":"https://api.scryfall.com/cards/a4cf1827-2ce6-4592-b213-858dcb115ae0/rulings","prints_search_uri":"https://api.scryfall.com/cards/search?order=released&q=oracleid%3A6bbd0bf6-cb22-431f-be1b-e76febf68cfc&unique=prints","collector_number":"2","digital":false,"rarity":"mythic","card_back_id":"2d53e50e-77d5-4353-9dde-0b1832d850f6","artist":"Mike Burns","artist_ids":["184b88b1-b679-4907-91b0-b2f2a458ae20"],"illustration_id":"365903b7-d641-4772-93ca-c1e5a0eca292","border_color":"black","frame":"2015","full_art":false,"textless":false,"booster":false,"story_spotlight":false,"prices":{"usd":null,"usd_foil":null,"eur":null,"tix":null},"related_uris":{"tcgplayer_decks":"https://decks.tcgplayer.com/magic/deck/search?contains=Dungeon+Master&page=1&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","edhrec":"https://edhrec.com/route/?cc=Dungeon+Master","mtgtop8":"https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Dungeon+Master"},"purchase_uris":{"tcgplayer":"https://shop.tcgplayer.com/productcatalog/product/show?ProductName=Dungeon+Master&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","cardmarket":"https://www.cardmarket.com/en/Magic/Products/Singles/Heroes-of-the-Realm-Promos/Dungeon-Master?referrer=scryfall&utm_campaign=card_prices&utm_medium=text&utm_source=scryfall","cardhoarder":"https://www.cardhoarder.com/cards?affiliate_id=scryfall&data%5Bsearch%5D=Dungeon+Master&ref=card-profile&utm_campaign=affiliate&utm_medium=card&utm_source=scryfall"},"_id":"Dungeon Master","_rev":"1-bee88cce49934d0d9ba2a4df9e0274f9"}},{"id":"Flesh // Blood","key":"Flesh // Blood","value":{"deleted":false,"rev":"1-9cf56363f743470cb33639f9f25ed300"},"doc":{"object":"card","id":"02b40fe4-901a-4832-8d52-a6bb5cc07b63","oracle_id":"0741151a-fec0-4ed3-9295-0d0f56d24165","multiverse_ids":[368991],"mtgo_id":48514,"tcgplayer_id":67919,"name":"Flesh // Blood","lang":"en","released_at":"2013-05-03","uri":"https://api.scryfall.com/cards/02b40fe4-901a-4832-8d52-a6bb5cc07b63","scryfall_uri":"https://scryfall.com/card/dgm/128/flesh-blood?utm_source=api","layout":"split","highres_image":true,"image_uris":{"small":"https://img.scryfall.com/cards/small/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.jpg?1562895503","normal":"https://img.scryfall.com/cards/normal/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.jpg?1562895503","large":"https://img.scryfall.com/cards/large/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.jpg?1562895503","png":"https://img.scryfall.com/cards/png/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.png?1562895503","art_crop":"https://img.scryfall.com/cards/art_crop/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.jpg?1562895503","border_crop":"https://img.scryfall.com/cards/border_crop/front/0/2/02b40fe4-901a-4832-8d52-a6bb5cc07b63.jpg?1562895503"},"mana_cost":"{3}{B}{G} // {R}{G}","cmc":7,"type_line":"Sorcery // Sorcery","colors":["B","G","R"],"color_identity":["B","G","R"],"card_faces":[{"object":"card_face","name":"Flesh","mana_cost":"{3}{B}{G}","type_line":"Sorcery","oracle_text":"Exile target creature card from a graveyard. Put X +1/+1 counters on target creature, where X is the power of the card you exiled.\nFuse (You may cast one or both halves of this card from your hand.)","watermark":"golgari","artist":"Lucas Graciano","artist_id":"ce98f39c-7cdd-47e6-a520-6c50443bb4c2","illustration_id":"5c13b10e-ae6b-4b28-89fa-16aa830f8267"},{"object":"card_face","name":"Blood","mana_cost":"{R}{G}","type_line":"Sorcery","oracle_text":"Target creature you control deals damage equal to its power to any target.\nFuse (You may cast one or both halves of this card from your hand.)","watermark":"gruul","artist":"Lucas Graciano","artist_id":"ce98f39c-7cdd-47e6-a520-6c50443bb4c2"}],"legalities":{"standard":"not_legal","future":"not_legal","historic":"not_legal","pioneer":"legal","modern":"legal","legacy":"legal","pauper":"not_legal","vintage":"legal","penny":"not_legal","commander":"legal","brawl":"not_legal","duel":"legal","oldschool":"not_legal"},"games":["mtgo","paper"],"reserved":false,"foil":true,"nonfoil":true,"oversized":false,"promo":false,"reprint":false,"variation":false,"set":"dgm","set_name":"Dragon's Maze","set_type":"expansion","set_uri":"https://api.scryfall.com/sets/c8bd8520-cd98-45cd-b533-8d40c2087426","set_search_uri":"https://api.scryfall.com/cards/search?order=set&q=e%3Adgm&unique=prints","scryfall_set_uri":"https://scryfall.com/sets/dgm?utm_source=api","rulings_uri":"https://api.scryfall.com/cards/02b40fe4-901a-4832-8d52-a6bb5cc07b63/rulings","prints_search_uri":"https://api.scryfall.com/cards/search?order=released&q=oracleid%3A0741151a-fec0-4ed3-9295-0d0f56d24165&unique=prints","collector_number":"128","digital":false,"rarity":"rare","card_back_id":"0aeebaf5-8c7d-4636-9e82-8c27447861f7","artist":"Lucas Graciano","artist_ids":["ce98f39c-7cdd-47e6-a520-6c50443bb4c2"],"illustration_id":"5c13b10e-ae6b-4b28-89fa-16aa830f8267","border_color":"black","frame":"2003","full_art":false,"textless":false,"booster":true,"story_spotlight":false,"edhrec_rank":19193,"prices":{"usd":"0.16","usd_foil":"0.68","eur":"0.23","tix":"0.02"},"related_uris":{"gatherer":"https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=368991","tcgplayer_decks":"https://decks.tcgplayer.com/magic/deck/search?contains=Flesh+%2F%2F+Blood&page=1&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","edhrec":"https://edhrec.com/route/?cc=Flesh+%2F%2F+Blood","mtgtop8":"https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Flesh+%2F%2F+Blood"},"purchase_uris":{"tcgplayer":"https://shop.tcgplayer.com/product/productsearch?id=67919&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","cardmarket":"https://www.cardmarket.com/en/Magic/Products/Singles/Dragons-Maze/Flesh-Blood?referrer=scryfall&utm_campaign=card_prices&utm_medium=text&utm_source=scryfall","cardhoarder":"https://www.cardhoarder.com/cards/48514?affiliate_id=scryfall&ref=card-profile&utm_campaign=affiliate&utm_medium=card&utm_source=scryfall"},"_id":"Flesh // Blood","_rev":"1-9cf56363f743470cb33639f9f25ed300"}},{"id":"Jace Beleren","key":"Jace Beleren","value":{"deleted":false,"rev":"1-23ea7953438c4bbf84be98316a535044"},"doc":{"object":"card","id":"97956927-135f-47b0-8aef-cc63889eab5a","oracle_id":"cc2b6e58-2d6f-495e-95f3-9496105c4cba","multiverse_ids":[393851],"tcgplayer_id":93796,"name":"Jace Beleren","lang":"en","released_at":"2014-12-05","uri":"https://api.scryfall.com/cards/97956927-135f-47b0-8aef-cc63889eab5a","scryfall_uri":"https://scryfall.com/card/jvc/1/jace-beleren?utm_source=api","layout":"normal","highres_image":true,"image_uris":{"small":"https://img.scryfall.com/cards/small/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.jpg?1562378464","normal":"https://img.scryfall.com/cards/normal/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.jpg?1562378464","large":"https://img.scryfall.com/cards/large/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.jpg?1562378464","png":"https://img.scryfall.com/cards/png/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.png?1562378464","art_crop":"https://img.scryfall.com/cards/art_crop/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.jpg?1562378464","border_crop":"https://img.scryfall.com/cards/border_crop/front/9/7/97956927-135f-47b0-8aef-cc63889eab5a.jpg?1562378464"},"mana_cost":"{1}{U}{U}","cmc":3,"type_line":"Legendary Planeswalker — Jace","oracle_text":"+2: Each player draws a card.\n−1: Target player draws a card.\n−10: Target player puts the top twenty cards of their library into their graveyard.","loyalty":"3","colors":["U"],"color_identity":["U"],"legalities":{"standard":"not_legal","future":"not_legal","historic":"not_legal","pioneer":"not_legal","modern":"legal","legacy":"legal","pauper":"not_legal","vintage":"legal","penny":"not_legal","commander":"legal","brawl":"not_legal","duel":"legal","oldschool":"not_legal"},"games":["paper"],"reserved":false,"foil":true,"nonfoil":false,"oversized":false,"promo":false,"reprint":true,"variation":false,"set":"jvc","set_name":"Duel Decks Anthology: Jace vs. Chandra","set_type":"duel_deck","set_uri":"https://api.scryfall.com/sets/37832684-2fe0-4b06-842b-eec06e5a17cb","set_search_uri":"https://api.scryfall.com/cards/search?order=set&q=e%3Ajvc&unique=prints","scryfall_set_uri":"https://scryfall.com/sets/jvc?utm_source=api","rulings_uri":"https://api.scryfall.com/cards/97956927-135f-47b0-8aef-cc63889eab5a/rulings","prints_search_uri":"https://api.scryfall.com/cards/search?order=released&q=oracleid%3Acc2b6e58-2d6f-495e-95f3-9496105c4cba&unique=prints","collector_number":"1","digital":false,"rarity":"mythic","card_back_id":"0aeebaf5-8c7d-4636-9e82-8c27447861f7","artist":"Kev Walker","artist_ids":["f366a0ee-a0cd-466d-ba6a-90058c7a31a6"],"illustration_id":"00146337-02df-4b1e-976b-8d64af966dd8","border_color":"black","frame":"2015","full_art":false,"textless":false,"booster":false,"story_spotlight":false,"edhrec_rank":866,"prices":{"usd":null,"usd_foil":"3.89","eur":null,"tix":null},"related_uris":{"gatherer":"https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=393851","tcgplayer_decks":"https://decks.tcgplayer.com/magic/deck/search?contains=Jace+Beleren&page=1&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","edhrec":"https://edhrec.com/route/?cc=Jace+Beleren","mtgtop8":"https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Jace+Beleren"},"purchase_uris":{"tcgplayer":"https://shop.tcgplayer.com/product/productsearch?id=93796&partner=Scryfall&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall","cardmarket":"https://www.cardmarket.com/en/Magic/Products/Search?referrer=scryfall&searchString=Jace+Beleren&utm_campaign=card_prices&utm_medium=text&utm_source=scryfall","cardhoarder":"https://www.cardhoarder.com/cards?affiliate_id=scryfall&data%5Bsearch%5D=Jace+Beleren&ref=card-profile&utm_campaign=affiliate&utm_medium=card&utm_source=scryfall"},"_id":"Jace Beleren","_rev":"1-23ea7953438c4bbf84be98316a535044"}}]
  }
}
