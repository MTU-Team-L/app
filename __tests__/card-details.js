import React from 'react';
import {test, expect} from 'jest-without-globals';
import {render} from 'react-native-testing-library';
import CardDetails from '../src/scenes/card-details';
import navigationMock from '../__mocks__/navigation-mock';
import {cards} from '../__mocks__/seed-data.json';

test('renders correctly', async () => {
  const testCard = cards.rows[0].doc;

  const {toJSON} = await render(<CardDetails navigation={navigationMock} route={{params: testCard}}/>);

  expect(toJSON()).toMatchSnapshot();
});
