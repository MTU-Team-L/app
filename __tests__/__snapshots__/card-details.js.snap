// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "width": 750,
      }
    }
  >
    <View
      onLayout={[Function]}
    >
      <View
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "backgroundColor": "transparent",
                "position": "relative",
              }
            }
          >
            <Image
              onLoad={[Function]}
              source={
                Object {
                  "uri": "https://img.scryfall.com/cards/art_crop/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783",
                }
              }
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 200,
                    "width": 750,
                  },
                ]
              }
              testID="RNE__Image"
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
            />
            <View
              accessibilityElementsHidden={true}
              importantForAccessibility="no-hide-descendants"
              pointerEvents="none"
              style={
                Object {
                  "bottom": 0,
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#bdbdbd",
                    "height": 200,
                    "justifyContent": "center",
                    "width": 750,
                  }
                }
                testID="RNE__Image__placeholder"
              />
            </View>
            <View
              style={
                Object {
                  "height": 200,
                  "width": 750,
                }
              }
            />
          </View>
        </View>
      </View>
      <Modal
        hardwareAccelerated={false}
        onRequestClose={[Function]}
        transparent={true}
        visible={false}
      >
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 1334,
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "top": 0,
              "width": 750,
            }
          }
        />
        <View
          onMoveShouldSetResponder={[Function]}
          onMoveShouldSetResponderCapture={[Function]}
          onResponderEnd={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderReject={[Function]}
          onResponderRelease={[Function]}
          onResponderStart={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          onStartShouldSetResponderCapture={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "flex": 1,
              "height": 0,
              "justifyContent": "center",
              "left": 0,
              "position": "absolute",
              "top": 0,
              "width": 0,
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "backgroundColor": "transparent",
                "position": "relative",
              }
            }
          >
            <Image
              onLoad={[Function]}
              source={
                Object {
                  "uri": "https://img.scryfall.com/cards/art_crop/front/a/4/a4cf1827-2ce6-4592-b213-858dcb115ae0.jpg?1573039783",
                }
              }
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": 200,
                    "width": 750,
                  },
                ]
              }
              testID="RNE__Image"
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
            />
            <View
              accessibilityElementsHidden={true}
              importantForAccessibility="no-hide-descendants"
              pointerEvents="none"
              style={
                Object {
                  "bottom": 0,
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#bdbdbd",
                    "height": 200,
                    "justifyContent": "center",
                    "width": 750,
                  }
                }
                testID="RNE__Image__placeholder"
              />
            </View>
            <View
              style={
                Object {
                  "height": 200,
                  "width": 750,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "top": 0,
              "width": 750,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 35,
                  "lineHeight": 40,
                  "shadowColor": "black",
                  "shadowOffset": Object {
                    "height": 0,
                    "width": 0,
                  },
                  "shadowOpacity": 0.8,
                  "shadowRadius": 1.5,
                  "textAlign": "center",
                  "width": 40,
                }
              }
            >
              ×
            </Text>
          </View>
        </View>
      </Modal>
    </View>
    <View
      pointerEvents="none"
      style={
        Object {
          "backgroundColor": "rgba(0, 0, 0, 0.2)",
          "height": 200,
          "position": "absolute",
          "top": 0,
          "width": 750,
        }
      }
    />
    <Text
      pointerEvents="none"
      style={
        Object {
          "bottom": 0,
          "color": "white",
          "fontSize": 35,
          "padding": 15,
          "position": "absolute",
          "width": 750,
        }
      }
      theme={
        Object {
          "colors": Object {
            "disabled": "hsl(208, 8%, 90%)",
            "divider": "#bcbbc1",
            "error": "#ff190c",
            "grey0": "#393e42",
            "grey1": "#43484d",
            "grey2": "#5e6977",
            "grey3": "#86939e",
            "grey4": "#bdc6cf",
            "grey5": "#e1e8ee",
            "greyOutline": "#bbb",
            "platform": Object {
              "android": Object {
                "error": "#f44336",
                "primary": "#2196f3",
                "secondary": "#9C27B0",
                "success": "#4caf50",
                "warning": "#ffeb3b",
              },
              "ios": Object {
                "error": "#ff3b30",
                "primary": "#007aff",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
            },
            "primary": "#2089dc",
            "searchBg": "#303337",
            "secondary": "#8F0CE8",
            "success": "#52c41a",
            "warning": "#faad14",
          },
        }
      }
    >
      Dungeon Master
    </Text>
  </View>
  <View
    style={
      Object {
        "margin": 15,
      }
    }
  >
    <SvgMock
      height={30}
      width={30}
    />
  </View>
  <View
    style={
      Object {
        "margin": 15,
      }
    }
  >
    <SvgMock
      height={30}
      width={30}
    />
  </View>
  <Text
    style={
      Object {
        "margin": 15,
        "marginTop": 0,
      }
    }
    theme={
      Object {
        "colors": Object {
          "disabled": "hsl(208, 8%, 90%)",
          "divider": "#bcbbc1",
          "error": "#ff190c",
          "grey0": "#393e42",
          "grey1": "#43484d",
          "grey2": "#5e6977",
          "grey3": "#86939e",
          "grey4": "#bdc6cf",
          "grey5": "#e1e8ee",
          "greyOutline": "#bbb",
          "platform": Object {
            "android": Object {
              "error": "#f44336",
              "primary": "#2196f3",
              "secondary": "#9C27B0",
              "success": "#4caf50",
              "warning": "#ffeb3b",
            },
            "ios": Object {
              "error": "#ff3b30",
              "primary": "#007aff",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
          },
          "primary": "#2089dc",
          "searchBg": "#303337",
          "secondary": "#8F0CE8",
          "success": "#52c41a",
          "warning": "#faad14",
        },
      }
    }
  >
    +1: Target opponent creates a 1/1 black Skeleton creature token with "When this creature dies, each opponent gains 2 life."
+1: Roll a d20. If you roll a 1, skip your next turn. If you roll a 12 or higher, draw a card.
−6: You get an adventuring party. (Your party is a 3/3 red Fighter with first strike, a 1/1 white Cleric with lifelink, a 2/2 black Rogue with hexproof, and a 1/1 blue Wizard with flying.)
  </Text>
</View>
`;
